confluence/server/scio_search/src/main/java/com/askscio/atlassian_plugins/confluence/impl/ScioSearchServletFilter.java

```diff
@@ -134,9 +138,15 @@ public void doFilter(
 
     final URL visitUrl;
     try {
-      visitUrl =
-          new URL(
-              String.format("%s%s?%s", baseURL, httpreq.getRequestURI(), httpreq.getQueryString()));
+      String query = httpreq.getQueryString();
```

Comment:
should we include the http method as a query param in case we need it in future?

Reply:
done.